GiEnJul/Startup.cs

```diff
@@ -28,8 +32,27 @@ public void ConfigureServices(IServiceCollection services)
             {
                 configuration.RootPath = "ClientApp/build";
             });
+            string domain = $"https://{Configuration["Auth0:Domain"]}/";
+            string audience = (_env.IsDevelopment()) ? $"https://{Configuration["Auth0:LocalAudience"]}/" : $"https://{Configuration["Auth0:AzureAudience"]}/";
```

Comment:
```suggestion
            string audience = $"https://{Configuration[(_env.IsDevelopment()) ? "Auth0:LocalAudience" : "Auth0:AzureAudience"]}/";
```
Could do something like this? 
Just preference I guess.

Reply:
Didn't know you could have the condition within the string